rotkehlchen/typing.py

```diff
@@ -354,6 +355,7 @@ class Location(DBEnumMixIn):
     GITCOIN = 30
     SUSHISWAP = 31
     SHAPESHIFT = 32
+    UPHOLD = 33
```

Comment:
you will need to add this to the DB schema too

Reply:
Isn't that what schema.py L88:60 does?

Reply:
yes. I must have missed it, or it was added after my initial review.